frontend/src/queries/schema.ts

```diff
@@ -388,7 +388,7 @@ export interface TimeToSeeDataSessionsQuery extends DataNode {
     /** Project to filter on. Defaults to current project */
     teamId?: number
 
-    response?: Record<string, any>[]
+    response?: { results: Record<string, any>[] }
```

Comment:
tip: declare it as a separate `TimeToSeeDataSessionsQueryResponse`. This will avoid `Response1` in python